### DISCUSSION TITLE
TL and PT Setup Support

### DISCUSSION THREAD

**Discussion by flatlineratheist** [egovernments/Digit-Core#610] [DIGIT Platform]
The current PT and LT setups are not running correctly.
The issue does not appear to be related to the database since we successfully created records using the same billing slabs in the last working instance. However, the records are not reflecting correctly this time.

**Comment by vinothrallapalli-eGov** [comment]
@flatlineratheist 
1) The current PT and LT setups are not running correctly. **What is the meaning of not running correctly? How can we guess in which part you are facing the issue? Please share the issue list and error logs or screenshots about issues.**

2) The issue does not appear to be related to the database since we successfully created records using the same billing slabs in the last working instance. However, the records are not reflecting correctly this time. **Please cross check if all services are up and running. If data was not persisting into DB then check the kafka is running and data was pushed into kafka, if not then fix the kafka issue. If data was pushed into kafka then check persister service is running and check persister logs if it is failing due to DB connectivity issues or data issues.**
